web/src/components/Korrel8rPanel.tsx

```diff
@@ -123,14 +126,7 @@ export default function Korrel8rPanel() {
         <Tooltip content={focusTip}>
           <Button
             isAriaDisabled={!korrel8rQueryFromURL}
```

Comment:
Basically the real issue is that by clicking the button over and over the panel makes repeated requests to korrel8r then displays them over and over one at a time. This change should prevent the user from making those repeat requests, which shouldn't be needed since the query already is the query from the URL.

Reply:
True, but the state of the cluster can change, so this would make it impossible to refresh the panel when things change in the cluster or the console. Also I'm seeing the problem frequently (not always) at the first press of Focus so I don't think repeated updates is the problem.

Reply:
Don't worry about this then. Just check out the other solution I had found